sfdc/modules/bulk/client.bal

```diff
@@ -18,25 +18,30 @@ import ballerina/http;
 import ballerina/io;
 import ballerinax/sfdc;
 
-# The Salesforce Bulk API Client object.
-#
+# Ballerina Salesforce connector provides the capability to access Salesforce Bulk API.
+# This connector lets you to perform bulk data operations for CSV, JSON, and XML data types.
+# 
 # + salesforceClient - OAuth2 client endpoint
 # + clientHandler - http:ClientOAuth2Handler class instance 
 # + clientConfig - Configurations required to initialize the `Client`
 @display {
     label: "Salesforce Bulk API Client",
     iconPath: "SalesforceLogo.png"
 }
-public client class Client {
-    http:Client salesforceClient;
-    http:OAuth2RefreshTokenGrantConfig|http:BearerTokenConfig clientConfig;
-    http:ClientOAuth2Handler|http:ClientBearerTokenAuthHandler clientHandler;
+public isolated client class Client {
+    private final http:Client salesforceClient;
+    private final http:OAuth2RefreshTokenGrantConfig|http:BearerTokenConfig clientConfig;
+    private final http:ClientOAuth2Handler|http:ClientBearerTokenAuthHandler clientHandler;
 
-    # Initializes Salesforce Bulk API Client.
+    # Initializes the connector. During initialization you can pass either http:BearerTokenConfig if you have a bearer
```

Comment:
To initialize, you can pass either the http:BearerTokenConfig if you have a bearer